etc/Makefile

```
@@ -149,9 +149,19 @@ $1: ../.image-$1
 	@make clean-$1 TARGETS_$1
 	@mkdir -p ../build/docker/$1
 	@cp -r docker/$1/* $$(FILES_$1) ../build/docker/$1
+ifdef MULTIARCH_BUILD
+	@echo "starting buildx builder"
+	@echo "cleanup buildx builder $1"
```

Let's combine these since we're hiding the cleanup anyway...
```suggestion
	@echo "starting buildx builder for $1"
```